helper.php

```diff
@@ -338,6 +339,18 @@ protected function linkToSearch($tag, $ns = '') {
         return '<a href="' . hsc($this->getTagSearchURL($tag, $ns)) . '">' . $tag . '</a>';
     }
 
+    /**
+     * Get the link to a page for the given pid
+     *
+     * @param string $pid search for this tag
+     * @param string $ns  limit search to this namespace
+     *
+     * @return string
+     */
+    protected function linkToPage($pid, $ns = '') {
+        return html_wikilink($pid);
+    }
```

Comment:
This method seems unnecessary? Also `$ns` isn't used.

Reply:
indeed, this was unnecessary. removed